README.md

```diff
@@ -0,0 +1,48 @@
+# hsac-fitnesse-fixtures
+[![Build Status](https://travis-ci.org/fhoeben/hsac-fitnesse-fixtures.svg?branch=master)](https://travis-ci.org/fhoeben/hsac-fitnesse-fixtures) [![Maven Central](https://img.shields.io/maven-central/v/nl.hsac/hsac-fitnesse-fixtures.svg?maxAge=300)](http://search.maven.org/#search|ga|1|g:"nl.hsac")
+
+This project assists in testing (SOAP) web services and web applications by providing an application to define and run tests. To this end it contains a baseline installation of FitNesse (an acceptance testing wiki framework) and some FitNesse fixture (base) classes.
+
+The fixtures provided aim to assist in testing (SOAP) web services and web applications (using Selenium) minimizing the amount of (custom) Java code needed to define tests.
+
+The baseline Fitnesse installation offers the following features:
+* Ability to easily create a standalone (no JDK or Maven required) Fitnesse environment.
+* Run Fitnesse tests on a build server, reporting the results in both `JUnit XML` format and `HTML`.
+* Fitnesse installation for fixture developers containing:
+    - the fixture base classes (and Selenium drivers for `Chrome`, `Internet Explorer`, `Edge` and `PhantomJs`),
```

Comment:
These concepts seem a bit strange for code formatting. Maybe italics would be better?